docs/gallery/tutorials/advanced_api_usage.py

```diff
@@ -0,0 +1,413 @@
+"""
+Advanced API Usage
+=================
+
+This page will cover some more advanced examples of Alpa.
+"""
+
+###########################################
+# We first import libraries and create example model and train step functions.
+
+import flax.linen as nn
+import jax
+import jax.numpy as jnp
+import ray
+import optax
+
+import alpa
+from alpa import global_config, parallelize
+from alpa.device_mesh import DeviceCluster
+from alpa.model.bert_model import BertConfig, FlaxBertLayer
+from alpa.model.model_util import TrainState
+from alpa.util import count_communication_primitives, get_ray_namespace_str
+
+# launch the cluster
+ray.init(address="auto", namespace=get_ray_namespace_str(prefix="alpa-example"))
+jax.config.update('jax_platform_name', 'cpu')
```

Comment:
This is no longer necessary. Alpa automatically sets this in the constructor of `DeviceCluster`
https://github.com/alpa-projects/alpa/blob/7dffd20a59b1414c6dee88d04498edd59b861458/alpa/device_mesh.py#L1326